.envrc

```diff
@@ -0,0 +1 @@
+use flake
```

Comment:
Btw this is not `nix-direnv` unless the users set it though (programs.direnv.enable on NixOS does this): https://github.com/nix-community/nix-direnv?tab=readme-ov-file#direnv-source_url

It will still work fine for every direnv user, just with less features (gc protection, eval caching).